core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1465,7 +1464,7 @@ public void onReceive(Context context, Intent intent) {
             Log.d(TAG, "Received Auto Connection update: " + status);
             if (!isConnectedToCar) {
                 Log.d(TAG, "Car was unplugged during playback.");
-                pauseIfPauseOnDisconnect();
+                pauseIfPauseOnDisconnect("bluetooth");
```

Comment:
Could you please change this to an enum?

Reply:
ready for a review